[Docs] Quickly explain how to use nvm, per #1030.

xee5ch opened the pull request:
Briefly explain what nvm is and what it does, and how that is separate from the npm package manager, as others say the summary only shows in the Github UI.

Fixes #1030.

Commit:
[Docs] Quickly explain how to use nvm, per #1030.

Briefly explain what nvm is and what it does, and how that is separate
from the npm package manager, as others say the summary only shows in
the Github UI.

ajstein commented:
This is just me spit-balling.  I was looking into some simple things to get into and saw this bug in the issue tracker. Not sure what and where you had in mind, but I thought I would give it a shot.

ljharb commented on README.markdown:
```diff
@@ -1,5 +1,11 @@
 # Node Version Manager [![Build Status](https://travis-ci.org/creationix/nvm.svg?branch=master)][3]
 
+## Why? Node has a package manager, NPM, right?
```
`npm` should always be lowercased

ljharb commented on README.markdown:
```diff
@@ -1,5 +1,11 @@
 
+## Why? Node has a package manager, NPM, right?
+
+Yes, it does.  `nvm` manages different versions of the node interpreter and core libraries.  Node is powerful, but the power of the Node ecosystem is the thousands of user-contributed libraries.  If you want additional libraries and tools, you use `npm` for that.
```
Please only put one space after a period.

ajstein replied:
OK, will fix it up in  a sec.

ljharb commented:
Sure, this seems fine! Thanks for the contribution.

Commit:
[Docs] Update #1030 PR with ljharb's comments.

Clean up the PR with the following requests from @ljharb.

- Convert spacing between sentences from one two.
- Always refer to npm in lower case.

ljharb commented:
@xee5ch would you mind checking the "allow edits" checkbox on the RHS of the PR?

vincent89291 reviewed (commented):
Explicit
